yarn-project/circuits.js/package.json

```diff
@@ -22,11 +22,8 @@
     "clean": "rm -rf ./dest .tsbuildinfo",
     "formatting": "run -T prettier --check ./src && run -T eslint ./src",
     "formatting:fix": "run -T prettier -w ./src",
-    "test": "NODE_NO_WARNINGS=1 node --experimental-vm-modules $(yarn bin jest) --passWithNoTests"
+    "test": "NODE_NO_WARNINGS=1 node --experimental-vm-modules $(yarn bin jest) --passWithNoTests --testTimeout=15000"
   },
-  "inherits": [
-    "../package.common.json"
-  ],
```

Comment:
Needed to remove this `inherits` or `yarn prepare` in CI yells about this `testTimeout`. Better way to handle this @ludamad?

I was hoping we could get rid of this timeout option now that we are no longer using UltraPlonk, but seems we still need it :thinking:

Reply:
Note you can change the timeout on individual tests, just add the timeout in ms as another parameter to `it` after the test function.

Or you can also tweak the timeout in the `jest` section of the package.json (see config [here](https://jestjs.io/docs/configuration#testtimeout-number)), that way you can keep using the same test command and globally change the default.

Reply:
oooo thanks @spalladino! I think it might just be one test timing out in which case I'll change that single one in the typescript. :pray:

Reply:
Turns out we don't need the timeout bump anymore now that we aren't using ultraplonk :thinking: might need to add it back in at some point, but for now we don't need it.